splitter.py

```diff
@@ -225,57 +182,95 @@ def _launch_simc_commands(commands, is_last_stage):
             f.cancel()
         executor.shutdown(wait=False)
         raise
-    return False
 
 
-def _start_simulation(files_to_sim, player_profile, simtype, simtype_value, stage, is_last_stage, num_profiles):
+def _start_simulation(files_to_sim: List[str],
+                      player_profile: Profile,
+                      stage: int,
+                      is_last_stage: bool,
+                      target_error: Optional[float] = None,
+                      iterations: Optional[int] = None):
     output_time = "{:%Y-%m-%d_%H-%M-%S}".format(datetime.datetime.now())
-
-    # some minor progress-bar-initialization
-    amount_of_generated_splits = 0
-    for file in files_to_sim:
-        if file.endswith(".simc"):
-            amount_of_generated_splits += 1
+    # TODO: is this needed?
```

Comment:
I can remember there being a reason for that - unfortunately I haven't documented it. 

Changing line 243 should make this check here obsolete.